### ISSUE TITLE
join libreoffice community

### ISSUE THREAD

**Issue by techsupp82** [LibreCAD/LibreCAD#1062]
I'm going straight to the point, we need a good alternative to autocad, the project has progressed very slowly lately, allows it to be part of the LibreOffice community, with all the advantages, also imitate the UI ribbon of Autocad 2019

**Comment by MikePryadko** [comment]
@techsupp82 UI ribbon sucks! :smiling_imp: 
AutoCAD is not very bad software, but not very good: better look at SolidWorks or [Компас-3D](https://kompas.ru/) (Russian software, better than AutoCAD, but a little bit worse than SolidWorks) :relieved:

**Comment by natrius** [comment]
Well, luckily the UI ribbon is optional after all. Would sound like a good step and maybe would bring more attention and maybe developers to help?

**Comment by fitojb** [comment]
I don’t think, as a member of The Document Foundation, that it would be a good fit for this project (given its current governance model), but we can be friends and collaborate :)

**Comment by techsupp82** [comment]
hahaha
________________________________
De: Adolfo Jayme-Barrientos ***@***.***>
Enviado: lunes, 9 de enero de 2023 02:02 a. m.
Para: LibreCAD/LibreCAD ***@***.***>
CC: techsupp82 ***@***.***>; Mention ***@***.***>
Asunto: Re: [LibreCAD/LibreCAD] join libreoffice community (#1062)


I don’t think, as a member of The Document Foundation, that it would be a good fit for this project (given its current governance model), but we can be friends and collaborate :)

**Comment by natrius** [comment]
> governance model

Out of curiosity - what do you mean with that?

**Comment by techsupp82** [comment]
por curiosidad  = out of curiosity (usamos la misma expresión) (we use the same expresion)

many times many free software teams drown in their own ideologies, when the main goal should always have been to gain more users, if there are more users there will be more resources... like blender for example. I've been a friend of free software for decades, I'm just stating the obvious.

I'm from Cordoba, we make jokes, thanks for the sense of humor

________________________________
De: Stefan ***@***.***>
Enviado: lunes, 9 de enero de 2023 12:05 p. m.
Para: LibreCAD/LibreCAD ***@***.***>
CC: techsupp82 ***@***.***>; Mention ***@***.***>
Asunto: Re: [LibreCAD/LibreCAD] join libreoffice community (#1062)


governance model

Out of curiosity - what do you mean with that?

**Comment by fa201** [comment]
5 years, we did not join LibreOffice community.
Should we close this and provide a statement to explain ?

**Comment by techsupp82** [comment]
No,  As the project grows and gains more adoption, there is not just one path. Best of luck.
Thanks!

________________________________
De: Fabrice ***@***.***>
Enviado: miércoles, 9 de agosto de 2023 04:32 p. m.
Para: LibreCAD/LibreCAD ***@***.***>
Cc: techsupp82 ***@***.***>; Mention ***@***.***>
Asunto: Re: [LibreCAD/LibreCAD] join libreoffice community (#1062)


5 years, we did not join LibreOffice community.
Should we close this and provide a statement to explain ?
